packages/flutter/test/widgets/nested_scroll_view_test.dart

```diff
@@ -3331,6 +3331,62 @@ void main() {
       areCreateAndDispose,
     );
   });
+
+  group('SliverAppBar.stretch in NestedScrollView', () {
```

Comment:
Is this group needed? It looks like there is only one test.
More tests would probably be good to have though. Looking at the existing SliverAppBar.stretch tests may indicate some cases this should cover.

Reply:
I didn't perform any previous tests on SliverAppBar.stretch, so I'm creating a test group here. Although there is only one test currently, I believe that future tests related to SliverAppBar.stretch can be added to this group.

Reply:
There are many tests for SliverAppBar.stretch in https://github.com/flutter/flutter/blob/master/packages/flutter/test/widgets/slivers_appbar_stretch_test.dart

We should make sure those scenarios work in the NestedScrollView with SliverAppBar.stretch in the tests here.